packages/sit-onyx/src/utils/validity.spec.ts

```diff
@@ -4,7 +4,7 @@ import { getFirstInvalidType } from "./validity.js";
 const getTestState = (overrideData: Partial<ValidityState>): ValidityState => ({
   valid: false,
   badInput: false,
-  customError: false,
+  error: false,
```

Comment:
In the native ValidityState the error is still called `customError`, so it shouldn't be changed here :)

```suggestion
  customError: false,
```